src/test/java/uk/gov/pay/publicauth/it/PublicAuthRevokeTokenResourceIT.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.pay.publicauth.it;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import uk.gov.pay.publicauth.utils.DropwizardAppWithPostgresExtension;
+
+import java.util.List;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.http.ContentType.JSON;
+import static java.lang.String.format;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.core.Is.is;
+
+@ExtendWith(DropwizardAppWithPostgresExtension.class)
+public class PublicAuthRevokeTokenResourceIT {
+
+    private Integer applicationPort;
+
+    @BeforeEach
+    public void setup(Integer port) {
+        applicationPort = port;
+    }
+    
+    @Test
+    public void revokeAllTokensForAnAccountSuccessfully() {
```

Comment:
I know it shouldn't happen but it might be safest to have a sanity check that it only revokes the tokens for the correct account, and doesn't somehow go revoking every token in the database

Reply:
Fixed